src/adapters/MeetingsJSONAdapter.js

```diff
@@ -177,6 +177,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
           remoteAudio: null,
           remoteVideo: null,
           remoteShare: null,
+          state: MeetingState.LEFT,
```

Comment:
`EMPTY` seems more suited here. The `ID` won't be null anymore, I think, in case of a "left" meeting, because we'll keep the meeting object with its normal id and state=LEFT.

so the check could just read `if(ID)`, and the comment above should read "A falsy ID signifies that the meeting was not yet created, or is invalid"